### DISCUSSION TITLE
Website loading problem

### DISCUSSION THREAD

**Discussion by Akhil-77** [idom-team/idom#1047] [Problem] [answered]
Hi, I was just accessing the jupyter notebook provided in the README section but it shows error in loading the book.

**Comment by rmorshea** [comment]
This looks like an issue with mybinder.org - will report it, but I don't think there's much that can be done.

**Reply by rmorshea** [reply, marked as answer]
For the time being, use this link: https://notebooks.gesis.org/binder/v2/gh/reactive-python/reactpy-jupyter/HEAD?labpath=notebooks%2Fintroduction.ipynb

mybinder.org runs on donated resources and it seems like one of the providers may be down and causing issues:
<img width="734" alt="image" src="https://github.com/reactive-python/reactpy/assets/4041990/566f7a4d-23ad-42d7-87cc-720da4d8ea99">

**Reply by Akhil-77** [reply]
okay thanks :)
